<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>936</width>
    <height>714</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="toolTip">
   <string comment="ddd"/>
  </property>
  <property name="styleSheet">
   <string notr="true">QDial {
	background: yellow;
}

#BKTemp,#MLTTemp,#HLTTemp {
 background:black;
color:yellow;
}

#BKSet,#MLTSet,#HLTSet {
 background:black;
color:yellow;
}

QGroupBox {
background:lightgray;
}

#Reset {
background:red;
}


</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="Reset">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>620</y>
      <width>881</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>22</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Emergency Stop - Reset All Outputs</string>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>891</width>
      <height>581</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>PID</string>
     </attribute>
     <widget class="QGroupBox" name="TempGroup">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>881</width>
        <height>541</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">QGroupBox {
	border: 20px;
}</string>
      </property>
      <property name="title">
       <string>Temperature Control</string>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
      <widget class="QPushButton" name="toggleMLT">
       <property name="geometry">
        <rect>
         <x>480</x>
         <y>440</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>SET MLT</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>370</x>
         <y>40</y>
         <width>151</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Lucida Console</family>
         <pointsize>48</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>MLT</string>
       </property>
       <property name="scaledContents">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QDial" name="MLTdial">
       <property name="geometry">
        <rect>
         <x>410</x>
         <y>420</y>
         <width>50</width>
         <height>64</height>
        </rect>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="maximum">
        <number>75</number>
       </property>
       <property name="invertedAppearance">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QPushButton" name="toggleHLT">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>440</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>SET HLT</string>
       </property>
      </widget>
      <widget class="QDial" name="HLTdial">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>420</y>
         <width>50</width>
         <height>64</height>
        </rect>
       </property>
       <property name="maximum">
        <number>75</number>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>40</y>
         <width>151</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Lucida Console</family>
         <pointsize>48</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>HLT</string>
       </property>
       <property name="scaledContents">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QWidget" name="horizontalLayoutWidget_3">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>280</y>
         <width>831</width>
         <height>121</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLCDNumber" name="HLTSet">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="toolTip">
           <string comment="dfg" extracomment="dfg"/>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Plain</enum>
          </property>
          <property name="lineWidth">
           <number>3</number>
          </property>
          <property name="smallDecimalPoint">
           <bool>true</bool>
          </property>
          <property name="segmentStyle">
           <enum>QLCDNumber::Flat</enum>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_5">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLCDNumber" name="MLTSet">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="toolTip">
           <string comment="dfg" extracomment="dfg"/>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Plain</enum>
          </property>
          <property name="lineWidth">
           <number>3</number>
          </property>
          <property name="smallDecimalPoint">
           <bool>true</bool>
          </property>
          <property name="segmentStyle">
           <enum>QLCDNumber::Flat</enum>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_6">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLCDNumber" name="BKSet">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="toolTip">
           <string comment="dfg" extracomment="dfg"/>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Plain</enum>
          </property>
          <property name="lineWidth">
           <number>3</number>
          </property>
          <property name="smallDecimalPoint">
           <bool>true</bool>
          </property>
          <property name="segmentStyle">
           <enum>QLCDNumber::Flat</enum>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>690</x>
         <y>40</y>
         <width>151</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Lucida Console</family>
         <pointsize>48</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>BK</string>
       </property>
       <property name="scaledContents">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QWidget" name="horizontalLayoutWidget_2">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>160</y>
         <width>831</width>
         <height>121</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLCDNumber" name="HLTTemp">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="toolTip">
           <string comment="dfg" extracomment="dfg"/>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Plain</enum>
          </property>
          <property name="lineWidth">
           <number>3</number>
          </property>
          <property name="smallDecimalPoint">
           <bool>true</bool>
          </property>
          <property name="segmentStyle">
           <enum>QLCDNumber::Flat</enum>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLCDNumber" name="MLTTemp">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="toolTip">
           <string comment="dfg" extracomment="dfg"/>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Plain</enum>
          </property>
          <property name="lineWidth">
           <number>3</number>
          </property>
          <property name="smallDecimalPoint">
           <bool>true</bool>
          </property>
          <property name="segmentStyle">
           <enum>QLCDNumber::Flat</enum>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLCDNumber" name="BKTemp">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="toolTip">
           <string comment="dfg" extracomment="dfg"/>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Plain</enum>
          </property>
          <property name="lineWidth">
           <number>3</number>
          </property>
          <property name="smallDecimalPoint">
           <bool>true</bool>
          </property>
          <property name="segmentStyle">
           <enum>QLCDNumber::Flat</enum>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QPushButton" name="toggleBoil">
       <property name="geometry">
        <rect>
         <x>750</x>
         <y>440</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>START BOIL</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>120</y>
         <width>81</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Heating:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>320</x>
         <y>120</y>
         <width>81</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Heating:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>610</x>
         <y>120</y>
         <width>81</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Heating:</string>
       </property>
      </widget>
      <widget class="QLabel" name="HLTPower">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>120</y>
         <width>81</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>0%</string>
       </property>
      </widget>
      <widget class="QLabel" name="MLTPower">
       <property name="geometry">
        <rect>
         <x>400</x>
         <y>120</y>
         <width>81</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>0%</string>
       </property>
      </widget>
      <widget class="QLabel" name="BKPower">
       <property name="geometry">
        <rect>
         <x>690</x>
         <y>120</y>
         <width>81</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>0%</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Program</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>871</width>
        <height>531</height>
       </rect>
      </property>
      <property name="title">
       <string>Program Status</string>
      </property>
      <widget class="QTableView" name="tableView">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>851</width>
         <height>501</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Graphs</string>
     </attribute>
     <widget class="QGroupBox" name="temperatureBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>871</width>
        <height>551</height>
       </rect>
      </property>
      <property name="title">
       <string>Temperature vs Time</string>
      </property>
      <widget class="QWidget" name="verticalLayoutWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>831</width>
         <height>501</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="plotlayout"/>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>936</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
